### DISCUSSION TITLE
Cmd + z [May 18]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2159] [Discord Threads]
> 🙋‍♂️ **Mr.Raccoon**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Not yet available. its tracked in our git  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/976187965678174339)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
